include/libswiftnav/almanac.h

```diff
@@ -42,6 +42,26 @@ typedef struct {
   double acc[3]; /**< velocity of the GEO at time toe [m/s^2] */
 } almanac_xyz_t;
 
+
+/** Structure containing the GLONASS almanac for one satellite. */
+
+typedef struct {
+  double lambda;      /**< Longitude of the first ascending node of the orbit
```

Comment:
GPS calls this omega, but keep it as `lambda` to prevent confusion with the use of omega for argument of perigee in the GLONASS ICD